dbms/src/Functions/FunctionsConversion.h

```diff
@@ -1751,6 +1751,114 @@ class FunctionDateFormat : public IFunction
     }
 };
 
+class FunctionGetFormat : public IFunction
+{
+private:
+    static String get_format(const String & time_type, const String & location)
```

Comment:
```suggestion
    static String get_format(const StringRef & time_type, const StringRef & location)
```
And then we don't need to call `StringRef.toString()`